login-workflow/src/screens/ContactSupport/ContactSupport.tsx

```diff
@@ -0,0 +1,252 @@
+                    {...slotProps.cardHeader}
+                />
+            )}
+            {!hiddenElements.contactSupportContent && (
+                <CardContent
+                    sx={DialogContentStyles(theme)}
+                    className={cx(defaultClasses.cardContent, classes.cardContent)}
+                    component={slots.cardContent}
+                    {...CardContentProps}
+                    {...slotProps.cardContent}
+                >
+                    {ContactSupportContent ? (
+                        ContactSupportContent
+                    ) : (
+                        <>
+                            {!hiddenElements.contactSupportIcon && (
+                                <Box sx={{ fontSize: 70, mb: 4, textAlign: 'center' }}>{ContactSupportIcon}</Box>
+                            )}
+                            {!hiddenElements.contactSupportQuestion && (
+                                <Typography variant={'body1'} sx={{ mb: 1 }}>
+                                    {contactSupportQuestion}
+                                </Typography>
+                            )}
+                            {!hiddenElements.contactSupportMessage && (
+                                <Typography>
+                                    {contactSupportMessage}
+                                    <Box component="a" href={`mailto:${emailId}`} sx={LinkStyles}>
+                                        {emailId}
+                                    </Box>
+                                    .
+                                </Typography>
+                            )}
+                            {!hiddenElements.contactSupportEmergency && (
+                                <Typography variant={'body1'} sx={{ mt: 4, mb: 1 }}>
+                                    {contactSupportEmergency}
+                                </Typography>
+                            )}
+                            {!hiddenElements.contactSupportTechnicalAssistance && (
+                                <Typography>
+                                    {contactSupportTechnicalAssistance}
+                                    <Box component="a" href={`tel:${phoneNumber}`} sx={LinkStyles}>
+                                        {phoneNumber}
+                                    </Box>
+                                    .
+                                </Typography>
+                            )}
+                        </>
+                    )}
+                </CardContent>
+            )}
+            {divider && <Divider />}
+            {!hiddenElements.actions && (
+                <CardActions
+                    sx={DialogActionsStyles(theme)}
+                    className={cx(defaultClasses.cardActions, classes.cardActions)}
+                    {...CardActionsProps}
+                >
+                    {Actions}
+                </CardActions>
+            )}
+        </>
+    );
+
+    return showInCard ? (
```

Comment:
can you get rid of this `showInCard` property? Let's just create "subscreens" for now. I heard that this is Joe's idea, but after discussions with Tom during PM OH, I think it is better to write "subscreens" instead of full screens.

Reply:
And please work with Manoj to get the file folder structure aligned